Goobi/config/goobi_config.properties

```diff
@@ -115,20 +115,24 @@ useOrigFolder=true
 # to have sufficient permissions to do whatever they should do.
 #
 # For examples of these scripts, see:
-# https://github.com/kitodo/kitodo-production/tree/1.11.x/Goobi/scripts
+#
+# https://github.com/kitodo/kitodo-production/tree/2.0.0/Goobi/scripts
+#
+# for script files of release 2.0.0. Never versions could be found under
+# appropriate release tag.
```

Comment:
Your suggestion for distribution and using this script files are good and should be regard in future development.